lib/Conversion/TorchToArith/TorchToArith.cpp

```diff
@@ -140,6 +140,25 @@ class ConvertAtenDivIntOp : public OpConversionPattern<AtenDivIntOp> {
 };
 } // namespace
 
+namespace {
+class ConvertAtenRemainderIntOp : public OpConversionPattern<AtenRemainderIntOp> {
```

Comment:
Drive by comment: this op also needs an e2e test. See: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

> DO use an [end-to-end test](https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md) if you are implementing a new op or extending the support for an existing op.